backend/tournesol/views/ratings.py

```diff
@@ -33,4 +42,15 @@ class ContributorRatingList(generics.ListAPIView):
     queryset = ContributorRating.objects.none()
 
     def get_queryset(self):
-        return ContributorRating.objects.filter(user=self.request.user)
+        ratings = ContributorRating.objects.filter(user=self.request.user)
+        is_public = self.request.query_params.get('is_public')
+        if is_public:
+            if is_public == "true":
+                ratings = ratings.filter(is_public=True)
+            elif is_public == "false":
+                ratings = ratings.filter(is_public=False)
+            else:
+                raise exceptions.ValidationError(
+                    "'is_public' query param should be 'true' or 'false'"
```

Comment:
We can enforce this valudation by using the word **must** instead of **should**.

Like:
> 'is_public' query param must be either 'true' or 'false'

Reply:
Done :+1: